src/components/ModalDelete/ModalDeleteComponent.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteNoteAction } from 'components/BottomBar/BottomBarActions';
+import { makeStyles } from '@material-ui/core';
+import { colors } from '../../colors';
+
+const useStyles = makeStyles(() => ({
+  dialog: {
+    color: 'white',
+    backgroundColor: colors.mainGray,
+  },
+  title: {
+    color: colors.white87,
```

Comment:
Please use `<Typography variant="subtitle1">` for the this text:

![image](https://user-images.githubusercontent.com/6686679/115607712-f79ad380-a2ed-11eb-92a1-2f3c9ed1cef0.png)

We have already defined styles for this text in [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L16)